www/src/components/PropsTable.jsx

```diff
@@ -69,7 +69,7 @@ const PropsTable = ({ props: componentProps, displayName, content }) => (
     {content && <div className="small mb-3">{content}</div>}
     {componentProps.length > 0 ? (
       <ul className="list-unstyled">
-        {componentProps.map(metadata => <Prop key={metadata.name} {...metadata} />)}
+        {componentProps.map(metadata => metadata.name !== 'intl' && <Prop key={metadata.name} {...metadata} />)}
```

Comment:
nit: think it's worth creating a `IGNORED_COMPONENT_PROPS=['intl']` (or similar) and use it in a `.filter(...)` to make this simple to add to in the future if needed?

Reply:
This definitely seems like a better approach, thanks 🙂